### ISSUE TITLE
Still Active

### ISSUE THREAD

**Issue by shawnbure** [joforce/JoforceCRM#90]
I don't want to be insensitive - so please don't take offense. Is this project still active? I have it installed and I really love the interface but I am getting errors and wanted to know if I should document them here?

**Comment by fenzik** [comment]
Hey @shawnbure

Yeah the project is active with many instance and we personally giving AMC support to 20+ customers 👍 

That will be great if you post the issues here, so we can priortize and fix those ASAP.

Thanks
Fenzik

**Comment by shawnbure** [comment]
Sure- good to hear it. I ram in to 2 separate issues that stopped my usage (I am evaluating yetiforce, vtiger community and joforce).

The GMAIL smtp config does not authenticate - just gives an error like it is not going through the process. I used the gmail option and custom option. (I have made 100% sure addresses and username and password are correct).

The workflows - attempting to delete workflow generates "Error" message and trying to create a new workflow generates "Error invalid expression".

No other logs or info than that.
